.github/workflows/publish.yml

```diff
@@ -0,0 +1,130 @@
+name: publish
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
```

Comment:
Are we sure on `pull_request`? Would this build and release the pr branch?

Reply:
oh good point. Let me remove that